src/main/scala/firrtl/stage/phases/Compiler.scala

```diff
@@ -111,8 +111,9 @@ class Compiler extends Phase with Translator[AnnotationSeq, Seq[CompilerRun]] {
       c.copy(stateOut = Some(annotationsOut))
     }
 
-    if (b.size <= 1) { b.map(f)         }
-    else             { b.par.map(f).seq }
+    if (b.size <= 1) { b.map(f) } else {
+      collection.parallel.immutable.ParVector(b :_*).par.map(f).seq
```

Comment:
`import scala.collection.parallel.CollectionConverters._` takes a better look and won't need to change this line?

Reply:
The problem is that `import scala.collection.parallel.CollectionConverters._` isn't available in `2.11`, `2.12` as far as I understand. See https://github.com/scala/scala-library-compat/issues/208